vllm/multimodal/utils.py

```diff
@@ -390,3 +393,36 @@ def modality_group_func(mm_input: MultiModalKwargs) -> Union[str, int]:
     return [
         list(group) for _, group in groupby(mm_inputs, key=modality_group_func)
     ]
+
+
+def run_dp_sharded_vision_model(image_input: torch.Tensor,
```

Comment:
we can add some unittest for function. (good for a follow up PR)

Reply:
@houseroad follow up PR is here https://github.com/vllm-project/vllm/pull/19103